### ISSUE TITLE
PHP 7.4 compatibility

### ISSUE THREAD

**Issue by gocom** [textile/php-textile#199]
Unit tests current do not work on PHP 7.4. These should be fixed. This may require dropping test support for old PHP versions or using workarounds for doing so, such as changing dependencies based on env.

### Prerequisites

* [x] Are you running the latest version of PHP-Textile?
* [x] `$ composer update`?

### Expected behaviour

Tests would pass without error.

### Actual behaviour

```
1) Netcarver\Textile\Test\BasicTest::testAdd with data set #10 ('fixtures/basic.yaml', 'Other Ruud stuff', array('*\n\n**\n\n* item1\n', '<p>*</p>\n\n<p>**</p>\n\n<ul>\n	<l...</ul>\n'))
Trying to access array offset on value of type bool
/home/travis/build/textile/php-textile/src/Netcarver/Textile/Parser.php:3091
/home/travis/build/textile/php-textile/src/Netcarver/Textile/Parser.php:2997
/home/travis/build/textile/php-textile/src/Netcarver/Textile/Parser.php:3596
/home/travis/build/textile/php-textile/src/Netcarver/Textile/Parser.php:3459
/home/travis/build/textile/php-textile/src/Netcarver/Textile/Parser.php:3312
/home/travis/build/textile/php-textile/src/Netcarver/Textile/Parser.php:1979
/home/travis/build/textile/php-textile/test/Netcarver/Textile/Test/BasicTest.php:61
/home/travis/build/textile/php-textile/vendor/phpunit/phpunit/phpunit:52
```

### Steps to reproduce

{Please write here}

#### Additional information

PHP-Textile version: master
PHP version: 7.4.0

**Comment by gocom** [comment]
Actually errors were coming from Textile parser itself. Was trying to access booleans as arrays.

**Comment by gocom** [comment]
Fixed in 67e6c02156fc991c24b2e042eaf25bc5aaa90dca
